CHANGELOG.md

```diff
@@ -0,0 +1,151 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.11.0] - 2022-06-10
+
+### Added
+
+- Message field assignments
+- Case expressions
+- Changelog
+
+## [0.10.0] - 2021-11-23
+
+### Added
+
+- `Byte_Order` aspect
+
+### Changed
+
+- Enable parallel build
+- Enable static build
+- Enable Ada API
+
+## [0.9.2] - 2021-11-24
+
+### Changed
+
+- Make install PEP 517 compliant to avoid fallback
+
+## [0.9.1] - 2021-11-10
+
+### Changed
+
+- Generate GPR file
+
+## [0.9.0] - 2021-10-08
+
+### Added
+
+- Type hints
+- Document process of updating `librflxlang.gpr`
+
+### Changed
+
+- Rename message components to fields
+
+## [0.8.1] - 2021-10-05
+
+### Changed
+
+- Build of source package
+
+### Changed
+
+- Rename package to `RecordFlux-parser`
```

Comment:
Here are two `Changed` sections

Reply:
Fixed. I have changed `Changed` to `Added`.